rpc/openapi/openapi.yaml

```diff
@@ -1666,6 +1666,9 @@ components:
                   codespace:
                     type: string
                     example: "ibc"
+            total_gas_used:
+              type: string
+              example: "100"
```

Comment:
shouldn't this be int?

Reply:
you are right. @crypto-facs want to make another quick PR?

Reply:
My bad! Sure I can make the change.. I do have one doubt though, if you notice all the other ints in that block response such as `gas_wanted` `gas_used` `height` are labeled as strings. Which is why I did the same. Do you know the reason for that? Maybe they are parse as string through the json formatter?

Reply:
Oh no it's a string. It's because we use a variant of proto json which encodes int64 as strings. My bad I was thinking plain json.

Reply:
Awesome that explains it! So Its good as it is 👌